.drone.yml

```diff
@@ -4656,6 +4656,55 @@ volumes:
 - name: dockersock
   temp: {}
 
+---
+################################################
+# Generated using dronegen, do not edit by hand!
+# Use 'make dronegen' to update.
+# Generated at dronegen/misc.go:30
+################################################
+
+kind: pipeline
+type: kubernetes
+name: publish-apt-rfd0058-repos
```

Comment:
We probably should not include RFD number in the pipeline name (I understand this may be a working name).

Reply:
I can remove it if you like, but it may make sense to leave it so that there is a clear differentiator between the new apt/yum repos, and the older repos. Do you have a suggestion for another way to make it clear to future maintainers what pipeline corresponds to what publishing system?

Reply:
This has been fixed